docs/ref/django-admin.txt

```diff
@@ -793,6 +793,17 @@ be warned that using ``--fake`` runs the risk of putting the migration state
 table into a state where manual recovery will be needed to make migrations
 run correctly.
 
+.. django-admin-option:: --plan
+
+Shows the order of unapplied migration operations Django will apply for the
+given ``migrate`` command. Unlike the similar ``showmigrations --plan``
```

Comment:
`showmigrations --plan` doesn't show the operations. (Perhaps worth making clear here.)

Reply:
I'll make that a bit more clear.